tests/functional/commands/validate/lib/models/policy_template_import_vaule_refs_parameters_with_same_name_as_policy_parameter.yaml

```diff
@@ -7,7 +7,7 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       Handler: index.handler
-      Runtime: nodejs16.x
+      Runtime: nodejs18.x
       CodeUri: s3://bucket/key
```

Comment:
If `s3://bucket/key` uses Lambda Provided JS SDK v2, this test will fail.

The JS SDK team recommends bumping to v3, and deploying the dependency in node_modules instead of depending on Lambda Provided one - which is provided for convenience.